portfolio/src/main/webapp/scripts/comments.js

```diff
@@ -59,6 +60,14 @@ function timestampToDateString(timestamp) {
   return dateString;
 }
 
+/**
+ * Repeats te given string the number given by rating.
```

Comment:
'te' should be 'the'.

[Optional] Also, since this function is generic and doesn't inherently have anything to do with stars maybe the function and parameter names should be made more generic?  It isn't even clear if the function is needed at all, since it seems like all it does is wrap the '.repeat' method.  Why not just call repeat directly instead of having this wrapper?

Reply:
I was thinking the same. After I wrote it I wasn't sure if I even needed it. I'll delete it.